easybuild/main.py

```diff
@@ -601,7 +604,7 @@ def process_eb_args(eb_args, eb_go, cfg_settings, modtool, testing, init_session
     success_msg = "Build succeeded "
     if build_option('ignore_test_failure'):
         success_msg += "(with --ignore-test-failure) "
-    success_msg += "for %s out of %s" % (correct_builds_cnt, len(ordered_ecs))
+    success_msg += f"for {correct_builds_cnt} out of {len(ordered_ecs)} in {time2str(datetime.now() - start_time)}"
```

Comment:
>  ==  Build succeeded for 5 out of 5 (total 46 mins 13 secs)

I was wondering how that is better than

>    == Build succeeded for 5 out of 5 in 46 mins 13 secs

but while that reads more fluently it has too many numbers next to each other to read in a quick glimpse and now I see better why you preferred the "...took" variant..

I'll add a colon too:

```suggestion
    success_msg += f"for {correct_builds_cnt} out of {len(ordered_ecs)} (total: {time2str(datetime.now() - start_time)})"
```